dot/network/discovery.go

```diff
@@ -233,6 +223,13 @@ func (d *discovery) findPeers() {
 	}
 }
 
-func (d *discovery) findPeer(peerID peer.ID) (peer.AddrInfo, error) {
-	return d.dht.FindPeer(d.ctx, peerID)
+func (d *discovery) stop() error {
```

Comment:
was there a reason to move this function?

Reply:
I have removed discovery) findPeer justt to make  service smaller, there is no reason to keep this as a function when we can call it directly. 
I moved stop to the very end to give logic to method sort, I usually like to have stop in the very end and start in the begging of the file. But I guess it is pretty subjective

Reply:
I like it! I am good with this change was just curious